tests/chaos/oadp/test_oadp.py

```diff
@@ -39,3 +40,38 @@ def test_reboot_vm_node_during_backup(
     oadp_backup_in_progress.wait_for_status(
         status=oadp_backup_in_progress.Backup.Status.PARTIALLYFAILED, timeout=TIMEOUT_10MIN
     )
+
+
+@pytest.mark.destructive
+@pytest.mark.chaos
+@pytest.mark.parametrize(
+    "rhel_vm_with_dv_running",
+    [
+        pytest.param(
+            {
+                "vm_name": "vm-node-drain-12020",
+                "rhel_image": RHEL_LATEST["image_name"],
+            },
+            marks=pytest.mark.polarion("CNV-12020"),
+        ),
+    ],
+    indirect=True,
+)
+def test_drain_vm_node_during_backup(
+    oadp_backup_in_progress,
+    drain_vm_source_node,
+):
+    """
+    Drain the worker node where the VM is located during OADP backup using DataMover.
+    Validate that backup eventually Completed.
+    """
+    jira_id = "CNV-67519"
+    if is_jira_open(jira_id):
+        oadp_backup_in_progress.wait_for_status(
+            status=oadp_backup_in_progress.Backup.Status.PARTIALLYFAILED, timeout=TIMEOUT_35MIN
+        )
+        pytest.xfail(f"Known bug {jira_id} still affects this test")
```

Comment:
I don't understand why need to run the test if you know that it has a bug. Spend 35 minutes just to confirm that the bug is still here?

I suppose, it should xfail immideately if the bug is still open

Reply:
I mean, if you know that there is a bug - why you need to run this test?